pkg/steps/template.go

```diff
@@ -46,6 +55,16 @@ func (s *templateExecutionStep) Inputs(ctx context.Context, dry bool) (api.Input
 	return nil, nil
 }
 
+func getOrgRepoFromVars(jobspec downwardapi.JobSpec) string {
+	vars := make(map[string]string)
+	envMap, _ := downwardapi.EnvForSpec(jobspec)
```

Comment:
I don't understand. Just do spec.refs.org. No need to create variables from it...

Reply:
let me address that :/

Reply:
addressed